articles/tutorials/building_2d_games/04_working_with_textures/index.md

```diff
@@ -0,0 +1,344 @@
+
+First, open the *Game1.cs* file in your project and add
+
+```cs
+private Texture2D _logo;
+```
+
+beneath where the [GraphicsDeviceManager](xref:Microsoft.Xna.Framework.GraphicsDeviceManager) and [SpriteBatch](xref:Microsoft.Xna.Framework.Graphics.SpriteBatch) instance member variables are declared.  This adds a new [Texture2D](xref:Microsoft.Xna.Framework.Graphics.Texture2D) instanced called `_logo`.  [Texture2D](xref:Microsoft.Xna.Framework.Graphics.Texture2D) is the type used to store a reference to 2D image data in MonoGame.
+
+Next, locate the [LoadContent](xref:Microsoft.Xna.Framework.Game.LoadContent) method and add the following after the `_spriteBatch` is instantiated:
+
+```
+_logo = Content.Load<Texture2D>("images/logo");
+```
+
+This uses the [ContentManager](xref:Microsoft.Xna.Framework.Content.ContentManager) object provided by the [Content](xref:Microsoft.Xna.Framework.Game.Content) property of the [Game](xref:Microsoft.Xna.Framework.Game) class to load the compiled image file.  We use the [Load<T>](xref:Microsoft.Xna.Framework.Content.ContentManager.Load``1(System.String)) method where `T` is the content type that we are loading, in this case a [Texture2D](xref:Microsoft.Xna.Framework.Graphics.Texture2D).  The parameter we pass to it is the *asset name*, which is a string representation of the path to the content to load, without any extensions.  The path used as the asset name is also relative to the value of [Content.RootDirectory](xref:Microsoft.Xna.Framework.Content.ContentManager.RootDirectory) which was set to `Content` in the `Game1` constructor.  THis means the asset name of our logo image located at *Content/images/logo.png* is `images/logo`.
+
+IF you run the game now, the image will be loaded as a texture, but all we'll see is the empty cornflower blue game window.  Next, we need to tell our game to draw the texture.
+
+## Drawing a Texture
+
+As mentioned in [Chapter 03](../03_the_game1_file/index.md#the-game-loop), all rendering should be done inside the [Draw](xref:Microsoft.Xna.Framework.Game.Draw(Microsoft.Xna.Framework.GameTime)) method. The [Draw](xref:Microsoft.Xna.Framework.Game.Draw(Microsoft.Xna.Framework.GameTime)) method's responsibility is to render the game state that was calculated in [Update](xref:Microsoft.Xna.Framework.Game.Update(Microsoft.Xna.Framework.GameTime))'; it should not contain any game logic or complex calculations.
+
+Locate the [Draw](xref:Microsoft.Xna.Framework.Game.Draw(Microsoft.Xna.Framework.GameTime)) method in the *Game1.cs* file and add the following after the [Clear](xref:Microsoft.Xna.Framework.Graphics.GraphicsDevice.Clear(Microsoft.Xna.Framework.Color)) method call is made:

```

Comment:
Call out elements like "Draw" and "Update" NEED to be bold or formatted in such a way as they stand out.  SO that the eye cannot miss them when reading.  E.G> Bold or encased in ``

Reply:
xref elements such as these have been marked as bold